blockchain/block/body.go

```diff
@@ -38,11 +38,11 @@ func (b *Body) Serialize() ([]byte, error) {
 }
 
 // LoadProto loads body from proto
-func (b *Body) LoadProto(pbBlock *iotextypes.BlockBody) error {
+func (b *Body) LoadProto(pbBlock *iotextypes.BlockBody, id uint32) error {
```

Comment:
With blockDeserializer module, body.LoadProto() and body.Deserializer() could be removed?